### PR TITLE
Metadata cleanup

### PR DESCRIPTION
**Pull request by limnoliver**
I did not add min/max for most things, but did run through the xml validation and fixed most errors. There are some publication or creation dates to fill in, and I've reached out to Farshid about these.

### CODE REVIEW

File: in_text/text_02_observations.yml [hunk 1]
```diff
@@ -69,4 +69,11 @@ entities:
...
       data-units: degrees Celsius
 
-file-format: comma-delimited files
+process-date: XX
+indirect-spatial: U.S.A.
+latitude-res: 0.00001
+longitude-res: 0.00001
+data-name: Flow and water temperature observations
+data-description: >-
+  Flow and water temperature observations used to train and validate models described in Rahmani et al. 2020.
```
Comment (aappling-usgs) [inline in_text/text_02_observations.yml]: Should we move flow observations to the Inputs item, too? (we already did this for flow predictions from the Ouyang model)

File: in_text/text_05_predictions.yml [hunk 2]
```diff
@@ -2,15 +2,15 @@ title: >-
...
     title: >-
-      Cross cite code base?
-    pubdate: XX
-    link: XX
+      Enhancing streamflow forecast and extracting insights using continental-scale long-short term memory networks with data integration
+    pubdate: 2020
+    link: https://doi.org/10.1029/2019WR026793
 
 build-environment: >-
   We used XX open source XX; Any supercomputing resources used? XX
```
Comment (aappling-usgs) [inline in_text/text_05_predictions.yml:16]: For this one, let's plan to (1) get more info from farshid about the compute environment (i added a bullet to "Text chunks we hope Farshid can fill in" in the "ERL data release plan" doc) and (2) add a reference to the environment.yml. I will make an issue for this so I can plan to do it - i have most of the info in hand already but will need a few minutes to put it together.

Comment (aappling-usgs) [reply in_text/text_05_predictions.yml:16]: (We could also just write one cover-everything text chunk that we use for all metadata files)

Comment (aappling-usgs) [reply in_text/text_05_predictions.yml:16]: added notes to https://github.com/USGS-R/rahmani_erl_data_release/issues/7

File: src/munge_functions.R [hunk 3]
```diff
@@ -22,7 +22,7 @@ extract_AQ_attributes <- function(out_file, in_file) {
 subset_pred_discharge <- function(out_file, in_file){
   dat <- readr::read_csv(in_file) %>%
     select(site_no, datetime, `sim_discharge(cfs)` = combine_discharge) %>%
-    filter(datetime >= as.Date('2013-09-30')) # predicted Q only in test period
+    filter(datetime > as.Date('2014-09-30')) # predicted Q only in test period
```
Comment (aappling-usgs) [inline src/munge_functions.R:25]: good catch!

**Review by aappling-usgs — CHANGES_REQUESTED** [review]
What do you think about moving the flow observations?

File: in_text/text_02_observations.yml (hunk 1 above)
Comment (limnoliver) [reply in_text/text_02_observations.yml]: Oh yeah, I did not move these. Will do.

File: src/munge_functions.R (hunk 3 above)
Comment (limnoliver) [reply src/munge_functions.R:25]: Farshid caught it! Glad he was looking at the files in detail :)

File: in_text/text_02_observations.yml (hunk 1 above)
Comment (limnoliver) [reply in_text/text_02_observations.yml]: Ok, done!

**Pull request merged**